crates/typst/src/geom/gradient.rs

```diff
@@ -157,13 +158,23 @@ use crate::syntax::{Span, Spanned};
 /// Typst predefines color maps that you can use with your gradients. See the
 /// [`color`]($color/#predefined-color-maps) documentation for more details.
 #[ty(scope)]
-#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+#[derive(Clone, PartialEq, Eq, Hash)]
 pub enum Gradient {
     Linear(Arc<LinearGradient>),
     Radial(Arc<RadialGradient>),
     Conic(Arc<ConicGradient>),
 }
 
+impl fmt::Debug for Gradient {
```

Comment:
Why this change? Since we split up Debug and Repr specifically to not have to override Rust's Debug style with the Typst repr.

Reply:
It was an artefact of me going frame-by-frame and transform-by-transform to figure out what was wrong, removed ;)